packages/jest-editor-support/src/Settings.js

```diff
@@ -33,18 +33,21 @@ type ConfigRepresentation = {
   testRegex: string,
   testMatch: Array<Glob>
 }
+import type {Options} from './types';
 
 module.exports = class Settings extends EventEmitter {
   debugprocess: ChildProcess;
-  workspace: ProjectWorkspace;
-
-  settings: ConfigRepresentation;
   jestVersionMajor: number | null;
+  options: Options;
+  settings: ConfigRepresentation;
+  workspace: ProjectWorkspace;
 
-  constructor(workspace: ProjectWorkspace) {
+  constructor(workspace: ProjectWorkspace, options?: Options) {
     super();
     this.workspace = workspace;
-
+    this.options = options || {
+      createProcess: jestChildProcessWithArgs,
+    };
```

Comment:
can you make this mirror the code in Runner.js with `this._createProcess`? Other than that it is ready to be merged.